packages/backend/src/functions/oauth/GitHub.ts

```diff
@@ -99,11 +100,22 @@ export default class GitHub {
 
     const user = await this.getUser(userId);
     if (isUser(user)) {
-      updateUser(filterUser(user));
+      if (await isFirstTime()) {
+        updateUser(filterUser(user));
+        unsetFirstTime();
+      } else if (await existUser(user)) {
```

Comment:
Since these cases do nothing now we should remove them

Reply:
What are you refering to? line 103 do something and 106 are you commenting below